policyengine_us/parameters/gov/states/co/ccap/entry_fpg_rate.yaml

```
@@ -0,0 +1,138 @@
+description: Colorado limits its Child Care Assistance Program application to filers with agi lower than these rates of household size federal poverty guidelines by county.
```

Spell out in descriptions, capitalize acronyms in labels
```suggestion
description: Colorado limits its Child Care Assistance Program application to filers with adjusted gross income lower than these rates of household size federal poverty guidelines by county.
```